src/app/models/Project.ts

```diff
@@ -33,17 +40,27 @@ export class Project extends AbstractModel implements IProject {
   public readonly kind: "Project" = "Project";
   public readonly id?: Id;
   public readonly name?: Param;
+  public readonly description?: Description;
   public readonly imageUrl?: string;
-  public readonly siteIds?: Ids;
+  @HasOne<User, AccountService>(ACCOUNT_SERVICE)
   public readonly creatorId?: Id;
   public readonly createdAt?: DateTimeTimezone;
+  @HasOne<User, AccountService>(ACCOUNT_SERVICE)
   public readonly updaterId?: Id;
   public readonly updatedAt?: DateTimeTimezone;
+  @HasOne<User, AccountService>(ACCOUNT_SERVICE)
   public readonly ownerId?: Id;
-  public readonly description?: Description;
+  @HasMany<Site, ShallowSitesService>(SHALLOW_SITES_SERVICE)
+  public readonly siteIds?: Ids;
+
+  // Associations
+  public readonly sites?: Observable<Site[]>;
```

Comment:
Why is this not `Observable<Site>`?

Reply:
Its an array of sites that gets returned